corehq/apps/hqadmin/tasks.py

```diff
@@ -180,3 +181,25 @@ def track_es_doc_counts():
                     }
                     metrics_gauge('elasticsearch.shards.docs.count', i['docs']['count'], tags)
                     metrics_gauge('elasticsearch.shards.docs.deleted', i['docs']['deleted'], tags)
+
+@periodic_task(queue='background_queue', run_every=crontab(minute="0", hour="0"))
+def track_pg_limits():
+    for db in settings.DATABASES:
+        with connections[db].cursor() as cursor:
+            cursor.execute("select table_name, data_type from information_schema.columns where column_name = 'id'")
+            query = """
+            select tab.relname, seq.relname, attlen
+              from pg_class seq 
+              join pg_depend dep on seq.oid=dep.objid 
+              join pg_class as tab on (dep.refobjid = tab.oid) 
+              join pg_attribute on attrelid=tab.oid and attnum=refobjsubid
```

Comment:
get column info for that column (size)

Reply:
nit: you could use `as att` here so that you could use `att.attlen`, etc., and not make people put it together that that's implicitly coming from this table. Not sure if there's a convention that goes against what I'm suggesting or something.